HACKING.md

```diff
@@ -210,6 +210,13 @@ oc label secret/azure-kola-tests-config \
     jenkins.io/credentials-type=secretFile
 oc annotate secret/azure-kola-tests-config \
     jenkins.io/credentials-description="Azure kola tests credentials config"
+
+oc create secret generic azure-managed-identity \
```

Comment:
Would you mind adding a comment above explaining the purpose of this secret? Or perhaps we should separate it into it's own `[OPTIONAL]` section. WDYT?

Reply:
Sure, I will make another optional config section, I think that fits best.

Reply:
Hmm, I added a comment to it; idk does nto quite fit the same form that was pre-existing wdyt? Maybe remove the ###Optional?